packages/imperative/src/imperative/src/plugins/utilities/npm-interface/uninstall.ts

```diff
@@ -19,8 +19,51 @@ import { ImperativeError } from "../../../../../error";
 import { ExecUtils, TextUtils } from "../../../../../utilities";
 import { StdioOptions } from "child_process";
 import { findNpmOnPath } from "../NpmFunctions";
+import { ConfigSchema, ProfileInfo } from "../../../../../config";
+import { IProfileTypeConfiguration } from "../../../../../profiles";
 const npmCmd = findNpmOnPath();
 
+/**
+ * Updates `extenders.json` and returns a list of types to remove from the schema, if applicable.
+ * @param npmPackage The package name for the plug-in that's being uninstalled
+ * @returns A list of types to remove from the schema
+ */
+export function updateAndGetRemovedTypes(npmPackage: string): string[] {
```

Comment:
If the intention is to make use of this function from outside imperative, we should modify the `npm-interface/index.ts` to include this new method in the list of exports 😋

Reply:
I exported this function for the sake of unit testing, but its not intended to be used outside of imperative. I've updated the style (to match the arrow function defined in `install.ts`) in commit 3ad57860.

Reply:
I exported this function for the sake of unit testing, but its not intended to be used outside of imperative. I've updated the style (to match the arrow function defined in install.ts) in commit 3ad57860.